### PR TITLE
process: add `Command::into_std()`

### PR DESCRIPTION
**Pull request by czy-29**
Fixes: #7001

### CODE REVIEW

File: tokio/src/process/mod.rs [hunk 1]
```diff
@@ -330,6 +330,11 @@ impl Command {
         &mut self.std
     }
 
+    /// Cheaply convert into a `std::process::Command`.
+    pub fn into_std(self) -> StdCommand {
```
Comment (Darksonn) [inline tokio/src/process/mod.rs:334]: We may want a note to say that Tokio-specific options may not apply to the underlying std command?

Comment (czy-29) [reply tokio/src/process/mod.rs]: Is this new version okay?

**Comment by czy-29** [comment]
The spell check failed... The error message appears at `kill_on_drop`, but this is a method name... I'm not sure how we should solve this spell check.

**Comment by Darksonn** [comment]
Method names must be enclosed in backticks. This ensures that the spell checker does not apply to them.
```diff
-/// Note that Tokio specific options (currently only option [kill_on_drop](Command::kill_on_drop))
+/// Note that Tokio specific options (currently only option [`kill_on_drop`](Command::kill_on_drop))
```

**Comment by czy-29** [comment]
Is it okay now?

**Review by ipetkov — APPROVED** [review]

File: tokio/src/process/mod.rs [hunk 2]
```diff
@@ -330,6 +330,14 @@ impl Command {
         &mut self.std
     }
 
+    /// Cheaply convert into a `std::process::Command`.
+    ///
+    /// Note that Tokio specific options (currently only option [`kill_on_drop`](Command::kill_on_drop))
+    /// will be lost.
```
Comment (Darksonn) [inline tokio/src/process/mod.rs:336]: This is a bit awkward to read. How about:
```suggestion
    /// Note that Tokio specific options will be lost. Currently, this only applies to [`kill_on_drop`].
    ///
    /// [`kill_on_drop`]: Command::kill_on_drop
```

Comment (czy-29) [reply tokio/src/process/mod.rs]: Yes, this is indeed a better document! Thank you for your suggestion!

**Comment by czy-29** [comment]
All modifications are completed, you can review and see if it can be merged.

**Comment by Darksonn** [comment]
Thank you.

**Pull request merged**